src/backend/distributed/operations/shard_rebalancer.c

```diff
@@ -2021,24 +2013,15 @@ GenerateTaskMoveDependencyList(PlacementUpdateEvent *move, int64 colocationId,
 
 /*
  * UpdateShardMoveDependencies function updates the dependency maps with the latest move's taskId.
+ * Currently there is just the nodeDependencies map to update. In the future we may add more.
```

Comment:
I do not think we need to comment on the code being future proof.

Reply:
To be honest I like future proof comments for 3 reasons:
- Comments don't really hurt - if it can give a clearer idea to why we have this current implementation to a new reader, why not add it?
- In the future, people who are totally new to this work may extend the current implementation. The comments might be useful for them.
- If we end up not doing any future work it's easier to clean up in the future if we have some future-proof comments available in the code. E.g. search by keyword "Future" or something